pkg/cmd/agent-task/create/create_test.go

```diff
@@ -151,10 +149,35 @@ func Test_createRun(t *testing.T) {
 		stubs            func(*httpmock.Registry)
 		baseRepoFunc     func() (ghrepo.Interface, error)
 		problemStatement string
+		baseBranch       string
 		wantStdout       string
 		wantStdErr       string
 		wantErr          string
 	}{
+		{
+			name:             "base branch included in create payload",
+			baseRepoFunc:     func() (ghrepo.Interface, error) { return ghrepo.New("OWNER", "REPO"), nil },
+			problemStatement: "Do the thing",
+			baseBranch:       "feature",
+			stubs: func(reg *httpmock.Registry) {
+				reg.Register(
+					httpmock.WithHost(httpmock.REST("POST", "agents/swe/v1/jobs/OWNER/REPO"), "api.githubcopilot.com"),
+					httpmock.RESTPayload(201, createdJobSuccessWithPRResponse, func(payload map[string]interface{}) {
+						prRaw, ok := payload["pull_request"].(map[string]interface{})
+						if !ok {
+							require.FailNow(t, "expected pull_request object in payload")
+						}
+						if prRaw["base_ref"] != "refs/heads/feature" {
+							require.FailNow(t, "expected pull_request.base_ref to be 'refs/heads/feature'")
+						}
+						if payload["problem_statement"] != "Do the thing" {
+							require.FailNow(t, "unexpected problem_statement value")
+						}
```

Comment:
Maybe simpler like this:

```suggestion
						require.True(t, ok, "expected pull_request object in payload")
						assert.Equal(t, "refs/heads/feature", prRaw["base_ref"], "expected pull_request.base_ref to be 'refs/heads/feature'")
						assert.Equal(t, "Do the thing", payload["problem_statement"], "unexpected problem_statement value")
```